internal/template/funcmap.go

```diff
@@ -58,6 +62,23 @@ func has(m map[string]interface{}, key string) bool {
 	return get(m, key) != nil
 }
 
+// hclField returns a hcl marshaled field e.g. `name = "foo"`
+func hclField(m map[string]interface{}, key string, req bool) (string, error) {
+	v, ok := m[key]
+	switch {
+	case ok:
+		b, err := hclencoder.Encode(v)
```

Comment:
`b` here can be a struct as well right?

Reply:
b are jsut bytes, I'm guessing you mean v? v can be any primitive or list or map

Reply:
right